src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Do you know why the linter gives an error?

Reply:
I've fixed it